test/runtime/Policies.go

```diff
@@ -1393,3 +1394,101 @@ var _ = Describe("RuntimeValidatedPolicyImportTests", func() {
 		Expect(res.Output().String()).Should(ContainSubstring(allowedVerdict), "Policy trace did not contain %s", allowedVerdict)
 	})
 })
+
+var _ = Describe("RuntimeValidatedPolicyDropAllTests", func() {
+	var once sync.Once
+	var logger *logrus.Entry
+	var vm *helpers.SSHMeta
+	var dropAll bool
+
+	initialize := func() {
```

Comment:
There is a reason for not adding the container creation in the BeforeEach function. The way the container is created is different i.e we need to create a docker container without any labels and also for egress we need to create a container without labels and start ping immediately on it. Hence this code cannot be common in `BeforeEach`

Reply:
This doesn't make sense to me, you can create a Context in the test and do the things that you have done in the test in there.

Reply:
maybe there is a miscommunication here. I will discuss this with @ianvernon and get back

Reply:
Still does not make sense to me. The `AfterAll` is a global method that we make sure that all is clean when it finish. That will not fail anyway. Test can be in a RuntimePolicies tests without having policies in multiple test groups and duplicated code.

Reply:
I pointed out the wrong. function:

```
BeforeAll(func() {
		****
		areEndpointsReady := vm.WaitEndpointsReady()
		Expect(areEndpointsReady).Should(BeTrue(), "Endpoints are not ready after timeout")
	})
```

I have tried this out and this will hang for the DROP_ALL tests. Also, we are unnecessary creating containers which this test does not need. I think we do not need to overkill the whole test-grouping process and group tests logically. Please note that drop_all is not a policy based test.

Reply:
@eloycoto : This is what I get if I try to incorporate this as part of `RuntimeValidatedPolicies`

```
   /Users/manalibhutiyani/go/src/github.com/cilium/cilium/test/ginkgo-ext/scopes.go:312

    Unable to delete all policies
    Expected
        <bool>: false
    to be true

    /Users/manalibhutiyani/go/src/github.com/cilium/cilium/test/runtime/Policies.go:441
------------------------------
RuntimeValidatedPolicies DROP_ALL Egress Policy test 
  DROP_ALL Egress Policy test

```

this is being hit in 
```
AfterEach(func() {
		vm.PolicyDelAll().ExpectSuccess("Unable to delete all policies")
	})
```
Since DROP_ALL test do not have any policies, this global `AfterEach` simply fails.